netlify.toml

```diff
@@ -1,13 +1,13 @@
 [build]
 publish = "public"
-command = "git submodule update --init --recursive --depth 1 && make production-build"
+command = "make production-build"
```

Comment:
Thanks for trying this. It might be working now because of the Netlify cache. I'm going to test that hypothesis by initiating a cache-clear-and-rebuild comment, and we can see how it goes.

Reply:
Makes sense!